### PR TITLE
pngs image: add image resources and add script for gen pngs.v

### PR DESCRIPTION
**Pull request by origel**
add image resources and add script for gen pngs.v. 

When add a new png image to project, just put png in images directory, and run `./gen_pngs.sh`, the `pngs.v` will be generated a new one.

### CODE REVIEW

**Comment by origel** [comment]
depend on PR#80 to resolve compile error.

**Comment by origel** [comment]
I add missing png image files( I get those files I from local file cache), and add a shell script for gen `pngs.v` file.

**Comment by origel** [comment]
@medvednikov any problems in this pull request? Take the time, give me a feedback. thanks.

**Comment by medvednikov** [comment]
Sorry for the delay.

I don't want to have pngs in the repository, at least not yet.

Also your arrow.png is a simple triangle, can't it be drawn with `gg`?

File: gen_pngs.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+v bin2v images/*.png -p bytes_ -m ui > pngs.v > pngs.v
```
Comment (spytheman) [inline gen_pngs.sh]: this should be just: `v bin2v images/*.png -p bytes_ -m ui > pngs.v`

**Comment by spytheman** [comment]
We can have a separate image repo as a submodule for https://github.com/vlang/ui , so that the pngs.v can be regenerated based on the images in the separate repository.

**Pull request closed without merging**